openwisp_firmware_upgrader/api/urls.py

```diff
@@ -0,0 +1,63 @@
+from django.conf.urls import url
+from django.urls import path
+from drf_yasg import openapi
+from drf_yasg.views import get_schema_view
+from openwisp_firmware_upgrader import settings as app_settings
+from rest_framework import permissions
+
+from . import views
+
+urlpatterns = []
+
+if app_settings.FIRMWARE_UPGRADER_API:
+
+    schema_view = get_schema_view(
+        openapi.Info(
+            title='Firmware Upgrader API',
+            default_version='v1',
+            description='OpenWISP Firmware Upgrader API',
+            # terms_of_service='http://openwisp.io/docs/',
+            contact=openapi.Contact(email='openwisp@googlegroups.com'),
+            license=openapi.License(name="GPLv3"),
+        ),
+        public=True,
+        permission_classes=(permissions.AllowAny,),
+    )
+
+    urlpatterns += [
+        # Swagger
+        url(
+            r'^swagger(?P<format>\.json|\.yaml)$',
+            schema_view.without_ui(cache_timeout=0),
+            name='schema-json',
+        ),
+        url(
+            r'^swagger/$',
+            schema_view.with_ui('swagger', cache_timeout=0),
+            name='schema-swagger-ui',
+        ),
+        url(
+            r'^redoc/$',
+            schema_view.with_ui('redoc', cache_timeout=0),
+            name='schema-redoc',
+        ),
+        # API endpoints
+        path('build/', views.build_list, name='api-build-list'),
+        path('build/<pk>/', views.build_detail, name='api-build-detail'),
+        path('firmware/', views.firmware_image_list, name='api-firmware-list',),
+        path(
+            'firmware/<pk>/', views.firmware_image_detail, name='api-firmware-detail',
```

Comment:
if we can be consistent with the previous requested change, hence having the following, it would be great:

```
build/<pk>/firmware-image/<pk>/
```